sdk/core/azure-core-test/src/main/java/com/azure/core/test/InterceptorManager.java

```diff
@@ -473,6 +473,22 @@ public void addSanitizers(List<TestProxySanitizer> testProxySanitizers) {
         }
     }
 
+    /**
+     * Disable common sanitizer rule for sanitization during record or playback.
+     * @param testProxySanitizerId the list of sanitizer rule Id to disable.
+     * @throws RuntimeException Neither playback or record has started.
+     */
+    public void removeSanitizers(List<String> testProxySanitizerId) {
+        if (CoreUtils.isNullOrEmpty(testProxySanitizerId)) {
+            return;
+        }
+        if (testProxyPlaybackClient != null) {
```

Comment:
@samvaity I think this is wrong. With this logic you can't remove sanitizers during recording.

Reply:
I don't think we need to remove sanitizers during recording, @scbedd is that right?

Reply:
I think we should. Otherwise the problematic sanitizers like `x-ms-encryption-key` header will be saved to disk with no useful information.